KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js

```diff
@@ -0,0 +1,489 @@
+define({ 
+
+devHeight :null,
+thisCard :null,
+thisCardIndex:null,
+cardFrameRel :null,
+
+frmAgendaPreshow : function(){
+  var self = this;
+  this.addActionToSessionTiles();
+  //this.view.tileOuterCopy.isVisible=false;
```

Comment:
Remove commented code

Reply:
Will discuss with Ed and will remove after the discussion